src/components/Header/index.jsx

```diff
@@ -1,13 +1,13 @@
 import React, { useState } from "react";
 import "./styles.scss";
-import headerBackground from "../../assets/images/bg_header.png";
+// import headerBackground from "../../assets/images/bg_header.png";
 import { Modal } from "../Modal";
 import { FormAddMovie } from "../FormAddMovie";
-import { useNavigate } from "react-router-dom";
+// import { useNavigate } from "react-router-dom";
```

Comment:
Plasea provide a router in your test suit to avoid commenting useNavigation hook:
`
import { MemoryRouter } from "react-router-dom";

describe("Card", () => {
  it("renders appropriately", () => {
    render(<MemoryRouter><Header /></MemoryRouter>);
    expect(screen.findAllByDisplayValue("FIND YOUR MOVIE"));
  });
});
`

See more information about testing react router: https://v5.reactrouter.com/web/guides/testing